va/va.h

```diff
@@ -2072,6 +2072,10 @@ typedef struct _VAPictureParameterBufferVC1
         uint32_t value;
     } transform_fields;
 
+    uint8_t luma_scale2;                  /* PICTURE_LAYER::LUMSCALE2 */
+    uint8_t luma_shift2;                  /* PICTURE_LAYER::LUMSHIFT2 */
+    uint8_t intensity_compensation_field; /* PICTURE_LAYER::INTCOMPFIELD */
```

Comment:
Is intensity_compensation_field used as index to table 109? Please add the comment if it is an index.